messages/en.json

```diff
@@ -12,5 +12,22 @@
   "store/checkout.shipping.shippingOptionsForAddressLabel": "Shipping options for",
   "store/checkout.shipping.continue": "Continue",
   "store/checkout.shipping.emptyDeliveryOptionsAlert": "No delivery options are available.",
-  "store/checkout.shipping.changeAddressLabel": "change"
+  "store/checkout.shipping.changeAddressLabel": "change",
+  "store/checkout.shipping.seeDetails": "See details",
+  "store/checkout.shipping.distance": "{distanceValue}km away",
```

Comment:
Can you confirm that this shouldn't be on imperial system? (miles, feets, etc instead of km)

Reply:
Nice catch! I think we need to convert the distance to miles, in this case.